src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,63 @@
+package mate.academy.service
+
+import mate.academy.model.Color
+import mate.academy.model.Figure
+import mate.academy.model.FigureType
+import mate.academy.model.impl.Circle
+import mate.academy.model.impl.IsoscelesTrapezoid
+import mate.academy.model.impl.Rectangle
+import mate.academy.model.impl.RightTriangle
+import mate.academy.model.impl.Square
+import kotlin.random.Random
+
+private const val MIN_SIZE: Double = 5.0
+private const val MAX_SIZE: Double = 15.0
+private const val DEFAULT_SIZE: Double = 10.0
+
+class FigureSupplier {
+    private val colorSupplier: ColorSupplier = ColorSupplier()
+
+    fun getDefaultFigure(): Figure = Circle(radius = DEFAULT_SIZE, color = Color.WHITE)
+    fun getRandomFigure(): Figure {
+        val randomColor: Color = colorSupplier.getRandomColor()
+        val randomIndexType: FigureType = FigureType.values()[Random.nextInt(FigureType.values().size - 1)]
```

Comment:
Could be simplified with FigureType.values().random().
Also please try to find better naming as this variable does not represent index